common/flatpak-utils.c

```diff
@@ -469,6 +469,45 @@ flatpak_get_arches (void)
   return (const char **) arches;
 }
 
+static char *
+get_os_release_value (const char *key,
+                      const char *default_value)
+{
+    const char *file = "/etc/os-release";
+    g_autofree char *contents = NULL;
+    g_autoptr(GKeyFile) keyfile = g_key_file_new ();
+
+    if (!g_file_test (file, G_FILE_TEST_EXISTS))
+        file = "/usr/lib/os-release";
+
+    if (!g_file_get_contents (file, &contents, NULL, NULL))
+        return g_strdup (default_value);
+
+    g_autoptr(GString) str = g_string_new (contents);
```

Comment:
We still declare all variables at the beginning of a block. (this happens at a bunch of places in this PR)